source/parser.hera

```diff
@@ -5581,15 +5610,31 @@ Reset
           children: [": { <B, A extends B> (a: A, b: B): b is A, <A, B> (a: A, b: B): a is A & B }"]
         }
         // [indent, statement]
-        module.prelude.push(["", ["const ", isRef, typeSuffix, " = Object.is", module.asAny, "\n"]])
+        module.prelude.push(["", ["const ", isRef, typeSuffix, " = Object.is", module.asAny, ";\n"]])
       },
       modulo(moduloRef) {
         const typeSuffix = {
           ts: true,
           children: [": (a: number, b: number) => number"]
         }
         // [indent, statement]
-        module.prelude.push(["", ["const ", moduloRef, typeSuffix, " = (a, b) => (a % b + b) % b", "\n"]])
+        module.prelude.push(["", ["const ", moduloRef, typeSuffix, " = (a, b) => (a % b + b) % b;", "\n"]])
+      },
+      xor(xorRef) {
+        const typeSuffix = {
+          ts: true,
+          children: [": (a: unknown, b: unknown) => boolean"]
+        }
+        // [indent, statement]
+        module.prelude.push(["", ["const ", xorRef, typeSuffix, " = (a, b) => a && b ? false : a || b;", "\n"]])
```

Comment:
For the future: it should be possible to get a much more accurate return type by using conditional types.